openapi/paths/invoices@{id}@transaction-allocations.yaml

```
@@ -5,16 +5,18 @@ get:
     - Core
   tags:
     - Invoices
-  summary: Get transaction amounts allocated to an invoice
+  summary: Retrieve transaction amounts allocated to an invoice
   operationId: GetInvoiceTransactionAllocationCollection
   x-sdk-operation-name: getAllTransactionAllocations
-  description: Get the precise amounts from a transaction allocated as invoice payments.
+  description: |-
+    Retrieves precise amounts from a transaction that were allocated
```

Nitpick: `precise` doesn't semm to add anything
```suggestion
    Retrieves amounts from a transaction that were allocated
```